evcouplings/utils/pipeline.py

```diff
@@ -472,6 +476,89 @@ def _handler(signal_, frame):
         raise
 
 
+def calculate_memory_requirements(config):
```

Comment:
This also has to take into account somehow how much memory jackhmmer may need (which can be *substantial*) - in case of long sequence the memory required for plm or mean-field dominates, but for shorter sequences with logs of homologs the size of the alignment might be a lot larger

Reply:
I don't see how this is possible to estimate unless you exactly know the data structures Jackhammer is using.

We could define a minimum RAM of 10G or so, to be on the save side?!

Reply:
Nope no way we can estimate this as we don't know how large sequence alignment is going to be - minimum value is definitely the way to go!